### PR TITLE
update modern-auth-for-office-2013-and-2016.md

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #364 ,
source [Updated Office 365 modern authentication](https://www.microsoft.com/microsoft-365/blog/2015/11/19/updated-office-365-modern-authentication-public-preview)

As the article's title is "How modern authentication works for Office 2013 and Office 2016 client apps", I didn't add Office 2010 in the table, I just added the "Note" to clarify that Office 2010 is not supported

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/modern-auth-for-office-2013-and-2016.md [hunk 1]
```diff
@@ -26,7 +26,10 @@ description: "Learn how Office 365 modern authentication works differently for O
 # How modern authentication works for Office 2013 and Office 2016 client apps
 
 Read this article to learn how Office 2013 and Office 2016 client apps use modern authentication features based on the authentication configuration on the Office 365 tenant for Exchange Online, SharePoint Online, and Skype for Business Online.
-  
+
+> [!NOTE]
+> Legacy client apps, such as Office 2010 or Office for Mac 2011, do not support modern authentication and could be used with basic authentication only.
```
Comment (JohanFreelancer9) [inline Enterprise/modern-auth-for-office-2013-and-2016.md]: Legacy client apps, such as Office 2010 **and** Office for Mac 2011, do not support modern authentication and **can only** be used with basic authentication.

Please do changes as indicated in bold.

Comment (nenonix) [reply Enterprise/modern-auth-for-office-2013-and-2016.md]: I think "or" is fine to use as it would be one or the other 😄

Comment (nenonix) [reply Enterprise/modern-auth-for-office-2013-and-2016.md]: **can only be used** with basic authentication.

**Comment by mypil** [comment]
@MaratMussabekov -please make the necessary correction based on the copy/edit review. Keep the "or" but just change this portion -  do not support modern authentication and **can only** be used with basic authentication.

Thanks.

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9 , your corrections were applied.
Thank you!

**Comment by mypil** [comment]
@MSFTTracyP - can you please assist to check, approve and merge the content for this PR based on issue #364?

Thank you.

**Comment by mypil** [comment]
@MSFTTracyP - I would like to follow up on this. Can you please assist to check, approve and merge the content for this PR based on issue #364?

Thank you.

**Comment by mypil** [comment]
@AndreaBarr - would you be able to merge the content of this PR? Thanks.

**Pull request merged**